requirements.txt

```diff
@@ -1,5 +1,5 @@
 dogapi==1.2.1
-django==1.8.13
+django==1.8.15
```

Comment:
Shouldn't this be using the same version the edx-platform uses?
Will this have an impact on the platform?

Reply:
1.8.15 is the same version which the current edx-platform uses. When you tried to install this XBlock, pip would have uninstalled the newer version and install 1.8.13. I made this change to prevent that.